### PR TITLE
Extra Pipes

### PR DESCRIPTION
**Pull request by TMRh20**
Finally found some time to play around with this and it was pretty easy to implement. Just a few questions now like whether to try and support mixed networks (I don't really like that idea) but it seems to work fine.

- Add support for any number of 'pipes' that are available for addressing. 
- Allows wider range of addressing on NRF52x devices

Instructions:
- Edit RF24Network_config.h and set NUM_PIPES to 8
- Edit RF24Mesh_config.h and set MESH_MAX_CHILDREN to 6
- Recompile and upload...

#201

### CODE REVIEW

File: RF24Network_config.h [hunk 1]
```diff
@@ -81,6 +81,12 @@
         /** Enable dynamic payloads - If using different types of nRF24L01 modules, some may be incompatible when using this feature **/
         #define ENABLE_DYNAMIC_PAYLOADS
     #endif // DISABLE_DYNAMIC_PAYLOADS
+    
+    /* The number of 'pipes' available for addressing in the current device
```
Comment (2bndy5) [inline RF24Network_config.h:85]:
```suggestion
    /** The number of 'pipes' available for addressing in the current device
```
For doxygen

File: RF24Network.cpp [hunk 2]
```diff
@@ -1181,7 +1181,7 @@ template<class radio_t>
 uint64_t ESBNetwork<radio_t>::pipe_address(uint16_t node, uint8_t pipe)
 {
 
-    static uint8_t address_translation[] = {0xc3, 0x3c, 0x33, 0xce, 0x3e, 0xe3, 0xec};
+    static uint8_t address_translation[] = {0xc3, 0x3c, 0x33, 0xce, 0x3e, 0xe3, 0xec, 0xee, 0xed};
```
Comment (2bndy5) [inline RF24Network.cpp:1184]:
```suggestion
    static uint8_t address_translation[] = {0xc3, 0x3c, 0x33, 0xce, 0x3e, 0xe3, 0xec
#if NUM_PIPES > 6
, 0xee
    #if NUM_PIPES > 7
, 0xed
    #endif
#endif
};
```
I wrote this from my phone, so clang-format will probably "fix" that suggestion.

**Review by 2bndy5 — CHANGES_REQUESTED** [review]
It's so simple and clean! 😃

**Comment by 2bndy5** [comment]
Citing a concern from #201 OP
> pipe/address validation will have to conditionally forego `x <= 5` presumption about 6 pipes

I think `is_valid_address()` needs tweaking:
https://github.com/nRF24/RF24Network/blob/a334fa6ec420b2beedaf617018c11847cea80402/RF24Network.cpp#L1134-L1138

**Comment by TMRh20** [comment]
I have it in the PR:
https://github.com/nRF24/RF24Network/pull/215/files#diff-8a8080552c7f675fa680e629d5c5baa2c875d8334cbd57d38b0e44bfc328b751R1134

**Review by 2bndy5 — APPROVED** [review]
oops! I didn't see that change's context... LGTM

**Pull request merged**